client/src/components/ProfileDetail/ProfileDetail.tsx

```diff
@@ -0,0 +1,57 @@
+import { Avatar, Card, CircularProgress, Grid, Paper, Typography } from '@mui/material';
+import { useAuth } from '../../context/useAuthContext';
+import React from 'react';
+import { useStyles } from './useStyles';
+import { LocationOn } from '@mui/icons-material';
+
+const ProfileDetail = (): JSX.Element => {
```

Comment:
The `ProfileDetails` page is specific to dog sitters, I don't think we would want to display the logged in user's profile in this page

Reply:
I used it as a placeholder for design FE, I will delete it.